database/db_connection.js

```diff
@@ -0,0 +1,24 @@
+const { Pool } = require('pg');
+const url = require('url');
+
+const env = require('env2');
+env('./config.env');
+
+if(!process.env.GROW_DB_URL) throw new Error('Environment variable GROW_DB_URL must be set');
+
+const params = url.parse(process.env.GROW_DB_URL);
+console.log(params);
+const [username, password] = params.auth.split(':');
+
+const options = {
+  host: params.hostname,
+  port: params.port,
+  database: params.pathname.split('/')[1],
+  max: process.env.DB_MAX_CONNECTIONS || 2
+};
+
+if (username) options.user = username;
```

Comment:
What is this if doing? In the case where username is undefined then option.user will be undefined with or without this if statement. To say that another way, The if statement has no effect on the outcome when username is undefined.

So what you are doing is testing for username being falsy.

Ie
```js
options.username = (username === '') ? undefined : username
```
would be the explicit (better) way of doing this (if this is what you are trying to do)